iree/compiler/Dialect/Flow/Transforms/FusionUtils.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+//===--- FusionUtils.cpp - Utilities that are useful for fusion
+//-------------===//
+//
+// Defines utility function and analyses that are useful across passes
```

Comment:
This comment is redundant since it is already in the header. (I don't know the convention, but duplication is not needed unless there is a convention.)